### ISSUE TITLE
linter-matlab does nothing (no errors)

### ISSUE THREAD

**Issue by jbrussell** [ajdm/linter-matlab#15]
I can't seem to get linter-matlab working. I installed but does nothing and throws no errors. The path to mlint is set to `/Applications/MATLAB_R2017a.app/bin/maci64/`. I have linter 2.3.1 installed.

**Comment by jbrussell** [comment]
If I put in a fake path to mlint it also does nothing and I get no errors... I'm at a loss.

**Comment by stephprince** [comment]
I have the same issue as well on both a Mac and PC. The relevant packages I have installed are linter-matlab 1.2.0, linter 2.3.1, linter-ui-default 1.8.0, intentions 1.1.5, and busy-signal 2.0.1. 

I've tried to set the mlintdir as Matlab2019b, 2019a, and 2018a. I also tried changing the linter-matlab.coffee file setting 'lintsOnChange' to true but that didn't do anything. I can see the linter display but it says 0 errors/warnings

**Comment by elpollodiablo84** [comment]
Finally found the problem!
You need to upgrade the linter version in the manifest file: look for the code (in `\linter-matlab\package.json`)
```
"providedServices": {
    "linter": {
      "versions": {
        "1.0.0": "provideLinter"
      }
    }
  }
```
and change the version from `"1.0.0"` to `"2.0.0"`.

**Comment by jdmanton** [comment]
Thanks! Fixed in v1.3.0.
